pkg/controllers/vscr/scrutinizepod_reconciler.go

```diff
@@ -144,6 +147,16 @@ func (s *ScrutinizeCmdArgs) buildScrutinizeCmdArgs() []string {
 		"--db-user", s.username,
 		"--hosts", strings.Join(s.hosts, ","),
 		"--log-path", paths.ScrutinizeLogFile,
+		"--tarball-name", s.tarballName,
 		"--honor-user-input",
 	}
 }
+
+// generateScrutinizeID returns a string, with the format VerticaScrutinize.yyyymmddhhmmss,
+// that will be used as the name of the tarball generated by scrutinize
+func generateScrutinizeID() string {
+	const idPrefix = "VerticaScrutinize."
+	const timeFmt = "20060102150405" // using fixed reference time from pkg 'time'
+	idSuffix := time.Now().Format(timeFmt)
```

Comment:
I think `time.Now()` returns the local time. Should we use UTC? Is that was scrutinize does?
```suggestion
	idSuffix := time.Now().UTC().Format(timeFmt)
```

Reply:
Yes this is the same function used in scrutinize.

Reply:
This is just a copy of the function in vclusterops scrutinize.

Reply:
I'm surprised by this. It could prevent ordering of these files if they were generated on separate machines.

Reply:
I spoke to Chris about this. He wasn't too worried either. No change needed here.